php/scripts/docker-entrypoint.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env sh
+
+set -e
+
+# See:
+# - Doc: https://docs.docker.com/engine/reference/builder/#entrypoint
+# - Example: https://github.com/docker-library/mariadb/blob/master/10.1/docker-entrypoint.sh
+#
+# Example use:
+# ./docker-entrypoint.sh php-fpm
+
+
+## Clear container on start by default
+if [ "$NO_FORCE_SF_CONTAINER_REFRESH" = "" ]; then
+    echo "Deleting container to make sure environment variables are picked up"
+    if [ -d ezpublish/cache ]; then
+        rm -f ezpublish/cache/*/*ProjectContainer.php
+    elif [ -d app/cache ]; then
+        rm -f app/cache/*/*ProjectContainer.php
+    elif [ -d var/cache ]; then
+        rm -f var/cache/*/*ProjectContainer.php
```

Comment:
can skip this as of symfony 3.2 where environment variables are supported and read at runtime _(new yml syntax for it)_.